Wikipedia/Code/TalkPageViewModel.swift

```diff
@@ -39,6 +39,7 @@ final class TalkPageViewModel {
     ///   - pageTitle: Wiki page title, e.g. "Talk:Cat" or "User_talk:Jimbo"
     ///   - siteURL: Site URL without article path, e.g. "https://en.wikipedia.org"
     ///   - articleSummaryController: article summary controller from the MWKDataStore singleton
+    ///   - authenticationManager: authentication manager from the MWKDataStore singleton
```

Comment:
Actually, I noticed the `dataController` parameter isn't used anywhere. It could be useful in the future for unit tests, but I decided to take it out for now.